extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
+1

as far as I remember we have enabled caching of any GET for 60 sec, could this be it?